### ISSUE TITLE
Python package dependencies

### ISSUE THREAD

**Issue by vreuter** [databio/dnameth_pipelines#12]
This could be more of a `pypiper` thing, but I'm opening it here since it's in the `wgbs.py` context that I'm encountering it. In the ``README` we nicely declare dependencies on a few of the `pepkit` and `databio` core packages, but do we track anywhere the 3rd-party ones? Specifically, where I ran into this was in one of the helper tools with a `biopython dependency`:
```
Traceback (most recent call last):
  File "/sfs/qumulo/qhome/vpr9v/code/dnameth_pipelines/src/tools/bisulfiteReadFiltering_forRNA.py", line 21, in <module>
    from Bio import SeqIO
ImportError: No module named Bio
</pre>
Process 191797 returned: (1). Elapsed: 0:00:01. Peak memory: (Process: 0.004GB; Pipeline: 22.502GB)
```

**Comment by nsheff** [comment]
No, we don't have this yet. This is basically the same issue as: https://github.com/databio/pypiper/issues/94

I'll flag that so it shows up there. Ultimately this could be solved by containers.

**Comment by vreuter** [comment]
- `Biopython`
- `bitarray`
- `guppy`
